openapi/nexus.json

```diff
@@ -8247,7 +8248,8 @@
           "system/ip-pools"
         ],
         "summary": "List IP pools",
-        "operationId": "ip_pool_list",
+        "description": "Operator endpoint to list all IP pools across the system.",
```

Comment:
Does this mean that it will list all IP pools across all silos? If so, maybe the description should clarify. It's not super obvious.

I'm not also a huge fan of the "Operator" term in user facing docs. It may not mean anything to our users, or it may mean something else to them. It really is up to each customer to define who has permissions to what, and it may not always fit neatly into what we think of as an "Operator"

Reply:
I think you're right on both counts. "List IP pools across all silos."